save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/cosv/CosvMetadata.kt

```diff
@@ -31,7 +30,6 @@ class CosvMetadata(
     var cosvId: String,
     var summary: String,
     var details: String,
-    var severity: String?,
```

Comment:
I think this change should affect database

Reply:
Fixed